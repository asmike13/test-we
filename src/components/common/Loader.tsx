import { Paper } from "@mui/material"
import HourglassBottomIcon from '@mui/icons-material/HourglassBottom';

const Loader = () => {
	return (
		<>
			<div className="overlay" />
			<Paper elevation={5} className="loader">
				<HourglassBottomIcon />
			</Paper>
		</>
	)
}

export default Loader
