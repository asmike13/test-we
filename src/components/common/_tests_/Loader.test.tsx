import renderer from 'react-test-renderer';
import Loader from '../Loader';

it('should render as snapshot', async () => {
	const snapshot = renderer.create(<Loader />)

	expect(snapshot).toMatchSnapshot()
})