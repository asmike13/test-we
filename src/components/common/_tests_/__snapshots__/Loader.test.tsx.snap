// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render as snapshot 1`] = `
Array [
  <div
    className="overlay"
  />,
  <div
    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation5 loader css-qh7kl5-MuiPaper-root"
  >
    <svg
      aria-hidden={true}
      className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
      data-testid="HourglassBottomIcon"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="m18 22-.01-6L14 12l3.99-4.01L18 2H6v6l4 4-4 3.99V22h12zM8 7.5V4h8v3.5l-4 4-4-4z"
      />
    </svg>
  </div>,
]
`;
